src/type/matrix/utils/broadcast.js

```diff
@@ -0,0 +1,84 @@
+import { factory } from '../../../utils/factory.js'
+
+const name = 'broadcast'
+
+const dependancies = ['concat']
+
+export const createBroadcast = /* #__PURE__ */ factory(
+  name, dependancies,
+  ({ concat }) => {
+    /**
+   * Broadcasts two matrices, and return both in an array
+   * It checks if it's possible with broadcasting rules
+   *
+   * @param {Matrix}   A      First Matrix
+   * @param {Matrix}   B      Second Matrix
+   *
+   * @return {Matrix[]}      [ broadcastedA, broadcastedB ]
+   */
+    return (A, B) => {
```

Comment:
This is a matter of personal preference, but can you change the anonymous function  `return (A, B) => { ... }` into return `return function broadcast (A, B) {...}` ? That will be more consistent with the code base.

Reply:
Yes of course. I didn’t notice that about the code base. I will change that.